src/prognostic_equations/hyperdiffusion.jl

```diff
@@ -113,9 +113,9 @@ NVTX.@annotate function prep_hyperdiffusion_tendency!(Yₜ, Y, p, t)
     @. ᶜ∇²specific_energy = wdivₕ(gradₕ(specific(Y.c.ρe_tot, Y.c.ρ) + ᶜp / Y.c.ρ - ᶜh_ref))
 
     if diffuse_tke
-        ᶜtke⁰ = @. lazy(specific(Y.c.sgs⁰.ρatke, Y.c.ρ))
+        ᶜtke = @. lazy(specific(Y.c.ρtke, Y.c.ρ))
         (; ᶜ∇²tke⁰) = p.hyperdiff
-        @. ᶜ∇²tke⁰ = wdivₕ(gradₕ(ᶜtke⁰))
+        @. ᶜ∇²tke⁰ = wdivₕ(gradₕ(ᶜtke))
```

Comment:
```suggestion
        @. ᶜ∇²tke = wdivₕ(gradₕ(ᶜtke))
```
Why not removing superscript 0 here and elsewhere?